rskj-core/src/main/java/org/ethereum/config/net/TestNetConfig.java

```diff
@@ -30,6 +31,7 @@ public TestNetConfig() {
         add(0, new TestNetBeforeBridgeSyncConfig());
         // 21 days of 1 block every 14 seconds.
         // On blockchain launch blocks will be faster until difficulty is adjusted to available hashing power.
-        add(129600, new TestNetAfterBridgeSyncConfig());
+        add(129_600, new TestNetAfterBridgeSyncConfig());
+        add(501_726, new TestNetDifficultyDropEnabledConfig());
```

Comment:
Why you added _ ?

Reply:
Because it improves the readability of the code